server/src/main/java/org/kaazing/gateway/server/context/resolve/DefaultConnectOptionsContext.java

```diff
@@ -40,10 +40,12 @@
 import org.kaazing.gateway.util.Utils;
 import org.kaazing.gateway.util.ssl.SslCipherSuites;
 import org.kaazing.gateway.util.ws.WebSocketWireProtocol;
+import static org.kaazing.gateway.service.TransportOptionNames.*;
 
 public class DefaultConnectOptionsContext implements ConnectOptionsContext {
 
     private static final long DEFAULT_WS_INACTIVITY_TIMEOUT_MILLIS = 0L;
+    private static int DEFAULT_HTTP_KEEPALIVE_TIMEOUT = 30; //seconds
```

Comment:
I forgot the final keyword, fixing it. The default is 30 secs (or could be lower) and can be configured via connect/accept option. We already have this option on the accept side. I see that some products (like nginx etc) offer it for configuration. But some (like JDK) don't offer it and manage behind the scenes.